cypress/pages/Finalization.js

```diff
@@ -0,0 +1,41 @@
+import { tid } from "../utils";
+import Summary from "./Summary";
+
+export default class Finalization {
+
+  constructor(trade) {
+    this.trade = trade;
+  }
+
+  willCreateProxy = () => {
```

Comment:
can we rename this methods from `willX` to `shouldX`? or `expectX` then it's clear that these are assertions. Otherwise I think it's not clear what they do